### PR TITLE
staff learner info card

### LINKED ISSUE #2530: 'Number of courses complete' for staff view of learner info
part of #2480

### PR DESCRIPTION
**Pull request by aliceriot**
#### What are the relevant tickets?
closes #2530 

#### What's this PR do?

Implements a card to show info about a learner's course stuff to a staff user.

(note: this is just the first part of the card, which incorporates part of the progress widget from the dashboard).

#### How should this be manually tested?

Make yourself a staff user on a program you're enrolled in, and visit your own learner page.

### CODE REVIEW

File: static/js/containers/LearnerPage_test.js [hunk 1]
```diff
@@ -59,9 +59,14 @@ import {
...
 } from '../util/test_utils';
+import StaffLearnerInfoCard from '../components/StaffLearnerInfoCard';
+import {
+  REQUEST_DASHBOARD,
+  RECEIVE_DASHBOARD_SUCCESS,
+} from '../actions/dashboard';
 
 describe("LearnerPage", function() {
-  this.timeout(10000);
+  this.timeout(15000);
```
Comment (rhysyngsun) [inline static/js/containers/LearnerPage_test.js]: We don't have to solve this here, but is there something in particular that is causing our tests here to run so long or is the LearnerPage just sufficiently complicated now that it takes this long to execute?

File: static/js/containers/LearnerPage_test.js [hunk 2]
```diff
@@ -982,6 +987,23 @@ describe("LearnerPage", function() {
...
+      return renderComponent(`/learner/${username}`, userActions).then(([wrapper, ]) => {
+        assert.equal(wrapper.find(StaffLearnerInfoCard).length, 0);
+      });
+    });
+
+
+    it('should show the staff learner info card, if the user has a staff role', () => {
+      const username = SETTINGS.user.username;
+      SETTINGS.roles.push({ role: 'staff' });
+      let actions = userActions.concat([REQUEST_DASHBOARD, RECEIVE_DASHBOARD_SUCCESS]);
```
Comment (rhysyngsun) [inline static/js/containers/LearnerPage_test.js]: Nit: could be `const`.

File: static/js/containers/LearnerPage_test.js (hunk 1 above)
Comment (rhysyngsun) [inline static/js/containers/LearnerPage_test.js]: Actually, looks like the answer is in Travis - tests failed because even 15s wasn't enough.

Comment (aliceriot) [inline static/js/containers/LearnerPage_test.js]: Yes, but I had an idea for a change that might make this unnecessary which I forgot to implement, I'll try it now.

**Comment by rhysyngsun** [comment]
Manual testing looks good too

**Pull request merged**